client/question/components/SpeechBalloonDirectives.js

```diff
@@ -0,0 +1,104 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directives for displaying speech balloons.
+ */
+
+tie.directive('tieSpeechBalloonContainer', ['$timeout', function($timeout) {
+  return {
+    restrict: 'E',
+    link: function(scope, iElement) {
+      var speechBalloonContainer = iElement[0];
+      speechBalloonContainer.className = "tie-speech-balloon-container";
```

Comment:
> Actually, both were not working. I would need to convert these to angular elements in order to use addClass. Is this required? Feels like it's making things more complicated for a theoretical situation I am not sure will happen.

Yes, we should still do this, I can see it breaking if we were to add e.g. protractor test classes.

Converting using angular.element() seems fine to me.

Reply:
Done